data/users.json

```diff
@@ -1,5 +1,6 @@
 [
   {
+    "pid": "1",
```

Comment:
For my information : Why now used a pid for user and not before this commit ?

Reply:
The `template` resource is the first to require direct links with the `patron` resource. You need to fix the creator of the template beforehand. Specially when you want to specify templates created by a system_librarian then you must specify the pid of the specific patron.